sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizePiiEntitiesTests.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Azure.AI.TextAnalytics.Tests
 {
-    [ServiceVersion(Min = TextAnalyticsClientOptions.ServiceVersion.V3_1)]
-    [Ignore("Not yet implemented")]
+    //[ServiceVersion(Min = TextAnalyticsClientOptions.ServiceVersion.V3_1)]
+    [ClientTestFixture(TextAnalyticsClientOptions.ServiceVersion.V2022_03_01_Preview)]
```

Comment:
Before adding more tests to other functionalities, I'm going to start working on enabling tests for all versions. This means I will delete enable `[ServiceVersion(Min = TextAnalyticsClientOptions.ServiceVersion.V3_1)]` and remove the `ClientTestFixture` from here